index.html

```diff
@@ -0,0 +1,309 @@
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile Portfolio</title>
+</head>
+
+<body>
+    <header class="header padding-left-right">
+        <a class="logo-link" href="#">
+            <h2 class="logo">TM</h2>
+        </a>
+        <nav class="main-nav">
+            <ul class="main-nav-list">
+                <li><a class="main-nav-link" href="#">Portfolio</a></li>
+                <li><a class="main-nav-link" href="#">About</a></li>
+                <li><a class="main-nav-link" href="#">Contact</a></li>
+                <li>
+                    <button class="mail-icon">
+<img src="img/icon-mail.svg" alt="Mail Icon" />
+</button>
+                </li>
+            </ul>
+        </nav>
+
+        <button class="btn-mobile-nav">
+<img class="menu-icon" src="img/menu.svg" alt="Hamburger menu icon" />
+</button>
+    </header>
+
+    <section class="headline-section padding-left-right">
+        <div class="headline-box">
+            <div class="headline-text-box">
+                <h1 id="headline-primary">
+                    Hey There. <br /> I'm Thato.
+                </h1>
+                <p class="headline-title">I am a software developer.</p>
+                <p class="headline-description">
+                    I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                </p>
+                <div class="social-icons">
+                    <ul class="social-icons-lists">
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-1.svg" alt="github social icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-2.svg" alt="linkedin social icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-3.svg" alt="pinterest social icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-4.svg" alt="twitter social icon" />
+                            </a>
+                        </li>
+
+                        <li>
+                            <a href="#">
+                                <img class="icon" src="img/icon-5.svg" alt="mercedes social icon" />
+                            </a>
+                        </li>
+                    </ul>
+                </div>
+            </div>
+
+            <div class="headline-img-box"></div>
```

Comment:
- [ ] In the headline section of the mobile view, you're also missing the background image as seen in the Figma design. So kindly fix this as well :+1: 
- [ ] Lastly, in the mobile view please ensure that there are no broken image links when on mobile view as well :wink: 

| Your Design | The Figma Design |
|--------|--------|
|![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/3d82ee3e-0506-4181-90ac-b2a2653d9f91) | ![image](https://github.com/Tarto-4/Desktop_Portfolio/assets/102699958/357a1db3-a203-4536-bbeb-fd86d8ccb1a6) |